### PR TITLE
refactor(styles): Remove bootstrap dependency on checkbox and radio

### PR DESCRIPTION
**Pull request by imagoiq**

### CODE REVIEW

**Comment by imagoiq** [comment]
Percy is green

File: packages/styles/src/mixins/_form-checks.scss [hunk 1]
```diff
@@ -2,48 +2,14 @@
...
-
-  .was-validated .form-check-input:invalid,
-  .form-check-input.is-invalid,
-  .was-validated .form-check-input:invalid:hover,
-  .form-check-input.is-invalid:hover {
-    &:not(:checked) {
-      border-color: color.$error;
-      background-color: color.$white;
+@mixin form-validation-state-selector($state) {
+  @if ($state == 'valid' or $state == 'invalid') {
```
Comment (alizedebray) [inline packages/styles/src/mixins/_form-checks.scss:6]: This condition is probably not necessary.
Also, this mixin is very generic and might be better in another file.

File: packages/styles/src/variables/components/_form-check.scss [hunk 2]
```diff
@@ -9,7 +9,7 @@
 $form-check-row-gap: spacing.$size-small-regular !default;
 $form-check-column-gap: spacing.$size-mini !default;
 $form-check-margin-bottom: spacing.$size-regular !default;
-$form-check-inline-margin-left: spacing.$size-large !default;
+$form-check-inline-margin-right: spacing.$size-large !default;
```
Comment (alizedebray) [inline packages/styles/src/variables/components/_form-check.scss:12]: Removing the "$form-check-inline-margin-left" is a breaking change, can you add the variable back to the deprecated variables in this file?

File: packages/styles/src/components/form-validation.scss [hunk 3]
```diff
@@ -39,51 +37,14 @@
...
-    ~ .#{$state}-feedback,
-    ~ .#{$state}-tooltip {
-      display: block;
-    }
-  }
-
-  .form-control {
-    @include form-validation-state-selector($state) {
-      padding-right: $input-height-inner;
-      background-image: escape-svg($icon);
```
Comment (alizedebray) [inline packages/styles/src/components/form-validation.scss:55]: The background should not be removed.

**Review by alizedebray — CHANGES_REQUESTED** [review]

**Comment by imagoiq** [comment]
Percy fails on alert. Issue is visible on next. Linked issue: https://github.com/swisspost/design-system/issues/2087

**Review by alizedebray — APPROVED** [review]

**Pull request merged**